.github/workflows/tests.yml

```diff
@@ -12,12 +12,12 @@ jobs:
     strategy:
       matrix:
         php: ['7.3', '7.4', '8.0', '8.1']
-        laravel: [5.5.*, 5.6.*, 5.7.*, 5.8.*, 6.*, 7.*, 8.*, 'dev-master as 9.0']
+        laravel: [5.5.*, 5.6.*, 5.7.*, 5.8.*, 6.*, 7.*, 8.*, 9.*]
```

Comment:
Fix tests with laravel dev-master, feel free to cherry pick the second commit has it has only this change or to fix it on main so I can rebase